### DISCUSSION TITLE
tflint reports error "Failed to initialize plugins; Plugin `aws` not found in /Users/user/.tflint.d/plugins"

### DISCUSSION THREAD

**Discussion by ameyaagashe** [terraform-linters/tflint#1753] [Q&A]
I have a few terraform files where I have intentionally added some error examples, incorrect instance type, and incorrect ami id.

Example below
```
resource "aws_instance" "ec2_instance" {
    ami = "ami-02396cdd13e9a1257"
    instance_type = "t2.xmicro"
} 
```
I expect **tflint** to report an error.

I have initialized the **tflint** and installed it with the commands below

```
brew install tflint
tflint --init
```

Unfortunately, when I run **tflint**, I get errors like the one below.

```
Failed to initialize plugins; Plugin `aws` not found in /Users/user/.tflint.d/plugins
```

I went to the Home directory and did a git clone of the was plugin and even created a symbolic link but it seems it is still not working :(

```
cd ~
mkdir -p .tflint.d/plugins
cd .tflint.d/plugins
cd tflint-ruleset-aws
git remote -v
https://github.com/terraform-linters/tflint-ruleset-aws.git

ls -alrth .tflint.d/plugins 
total 0
drwxr-xr-x@  3 user  staff    96B Apr 30 13:54 ..
drwxr-xr-x@ 20 user  staff   640B Apr 30 13:54 tflint-ruleset-aws
lrwxr-xr-x@  1 user  staff    18B Apr 30 13:55 aws -> tflint-ruleset-aws
drwxr-xr-x@  4 user  staff   128B Apr 30 13:55 .
```
Do you have any idea what the issue is and how I can resolve it?
P.S: My .tflint.hcl is as per below:
```
plugin "terraform" {
  enabled = true
  preset  = "recommended"
}
plugin "aws" {
  enabled = true
  deep_check = true
}
rule "aws_instance_invalid_type" {
  enabled = true
}
```

**Comment by bendrucker** [comment]
Plugins are compiled binaries so obtaining the source via `git clone` won't work. Do you get errors from `tflint --init`? What is your config?

**Reply by ameyaagashe** [reply]
Thank you Ben.
I will add .tflint.hcl in my OP shortly
